CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# New Features (In Progress)
+# dbt_quickbooks v0.6.1
```

Comment:
This should be a breaking change with everything going into the package. Even if it may not actually cause a breaking change, we should document it as such since it is quite large with the amount of features being added. Let's bump this to 0.7.0

Reply:
Updated